docs/fundamentals/runtime-libraries/snippets/System.Diagnostics/PerformanceCounter/NextValue/csharp/elapsedtime.cs

```diff
@@ -1,4 +1,4 @@
-﻿// Notice that the sample is conditionally compiled for Everett vs.
+// Notice that the sample is conditionally compiled for Everett vs.
```

Comment:
Really? We're talking about Everett and Whidbey?

Reply:
This PR was just a bulk change with a tool. I didn't update anything unless it was necessary to get the code to compile. Updating all the old stuff like this would require weeks of work!